### PR TITLE
Whitelist cyberark API and CLI repos

### PR DESCRIPTION
**Pull request by jtuttle**

### CODE REVIEW

File: lib/ws.rb [hunk 1]
```diff
@@ -9,13 +9,14 @@ class WS < ::Sinatra::Base
...
     "conjur-api" => "conjurinc/api-node",
+    "cyberark-conjur-api-node" => "cyberark/conjur-api-node"
   }
   
   GEM_REPOS = {
     "conjur-api" => "conjurinc/api-ruby",
     "conjur-cli" => "conjurinc/cli-ruby",
-    "cyberark-conjur-api" => "cyberark/conjur-api-ruby"
-    "cyberark-conjur-cli" => "cyberark/conjur-cli"
+    "cyberark-conjur-api" => "cyberark/conjur-api-ruby",
```
Comment (dustinmm80) [inline lib/ws.rb]: The key here shouldn't change: `conjur-api` stays, `conjurinc/api-ruby` changes to `cyberark/conjur-api-ruby`. 

Same for `conjur-cli`. We are releasing the gems under the same name, but the github repos have changed.

File: lib/ws.rb [hunk 2]
```diff
@@ -9,13 +9,14 @@ class WS < ::Sinatra::Base
   
   NPM_REPOS = {
     "conjur-api" => "conjurinc/api-node",
+    "cyberark-conjur-api-node" => "cyberark/conjur-api-node"
```
Comment (dustinmm80) [inline lib/ws.rb]: You can delete this line and make line 11

`"conjur-api" => "cyberark/conjur-api-node",`

That's what we want here.

Comment (jtuttle) [inline lib/ws.rb]: Ah okay, thought you said something about leaving the existing ones in place. Fixed!

File: lib/ws.rb (hunk 1 above)
Comment (jtuttle) [inline lib/ws.rb]: Done! But again I thought you said to leave the existing ones in place. Maybe you were just referring to the key. :)

**Pull request merged**